tpp-app/src/home/pages/LogSymptomsScreen.js

```diff
@@ -13,6 +13,7 @@ import { TRACK_SYMPTOMS } from "../../services/utils/constants";
 import { CALENDAR_STACK_SCREENS } from "../CalendarNavigator";
 import { getISODate } from '../../services/utils/helpers';
 import { calculateAverages } from "../../services/CalculationService";
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
```

Comment:
need to push package.json and package-lock.json as well

Reply:
Bruh why didn't they pushhh 

thx for catching it ><